ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -197,7 +197,7 @@ export const testMarkdownWhisper = (): Promise<boolean> =>
               | - | - |
               | Row 1 Col 1 | Row 1 Col 2 |
               | Row 2 Col 1 | Row 2 Col 2 |
```

Comment:
Should we remove tab space before each line in the markdown? Perhaps I didn't fully understood the issue.